Apps/Sandcastle/index.html

```diff
@@ -21,6 +21,7 @@
     <script src="jsHintOptions.js"></script>
     <script src="gallery/gallery-index.js"></script>
     <script src="CesiumSandcastle.js"></script>
+    <script src="../../Source/Cesium.js"></script>
```

Comment:
Actually I'm pretty sure Sandcastle already pulls in a copy of Cesium (likely the prebuilt copy) for code auto-complete purposes.  I think `Cesium.VERSION` should already be defined without this line.